components/ww3/src/ww3_utils.cmake

```diff
@@ -0,0 +1,116 @@
+# Set switch file on command line when running CMake
+#set(SWITCH "" CACHE STRING "Switch file, either full path, relative path from location of top-level WW3/ dir, or a switch in model/bin")
+
+function(parse_switches)
```

Comment:
Would it be possible to move any of this logic to python buildnml/buildlib_cmake? Cmake is very useful but it's not great as a general programming language.